src/Sylius/Bundle/ResourceBundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -46,6 +46,14 @@ public function load(array $config, ContainerBuilder $container)
             $loader->load($configFile);
         }
 
+        $container->setAlias('sylius.resource_controller.authorization_checker', 'sylius.resource_controller.authorization_checker.disabled');
+
+        if (class_exists('Sylius\Bundle\RbacBundle\SyliusRbacBundle')) {
```

Comment:
Checking the `SyliusRbacBundle` class exists is not enough and does not mean it is used/loaded.
You should use `$container->hasExtension(...)` and move this to `prepend` by implementing `PrependExtensionInterface`

```
    /**
     * {@inheritdoc}
     */
    public function prepend(ContainerBuilder $container)
    {
        if (!$container->hasExtension('sylius_rbac')) {
            $container->setAlias('sylius.resource_controller.authorization_checker', 'sylius.resource_controller.authorization_checker.disabled');

            return;
        }

        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
        $loader->load('rbac.xml');

        $container->setAlias('sylius.resource_controller.authorization_checker', 'sylius.resource_controller.authorization_checker.rbac');
    }
```